projects/packages/connection-ui/package.json

```
@@ -0,0 +1,39 @@
+{
+	"name": "jetpack-connection-manager-ui",
+	"version": "1.0.0",
+	"description": "Jetpack Connection Manager UI",
+	"main": "_inc/admin.jsx",
+	"repository": "https://github.com/Automattic/jetpack-connection-ui",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"scripts": {
+		"build": "yarn install-if-deps-outdated && yarn clean && yarn build-client",
+		"build-client": "gulp",
+		"clean": "rm -rf build/",
+		"install-if-deps-outdated": "yarn check 2> /dev/null || yarn install --check-files --production=false --frozen-lockfile",
```

```suggestion
		"install-if-deps-outdated": "yarn install --check-files --production=false --frozen-lockfile",
```

We can remove this. It was removed upstream from the main one, it's a deprecated check that doesn't do anything different than yarn install --check-files.